manager/controllers/package_versions.go

```diff
@@ -19,6 +19,7 @@ var PackageVersionsOpts = ListOpts{
 	// By default, we show only fresh systems. If all systems are required, you must pass in:true,false filter into the api
 	DefaultFilters: map[string]FilterData{},
 	DefaultSort:    "evra",
+	SecondarySort:  "evra",
```

Comment:
SecondarySort value should be a column with unique values.  In this case it joins package table so `package.id ` could be a good candidate.